web/handlers/http.go

```diff
@@ -195,7 +224,19 @@ func New(
 	adminHandler := a.Authenticate(admin.Handler(r, roomState, al, ns, ps))
 	mainMux.Handle("/admin/", adminHandler)
 
-	m.Get(router.CompleteIndex).Handler(r.StaticHTML("landing/index.tmpl"))
+	m.Get(router.CompleteIndex).Handler(r.HTML("landing/index.tmpl", func(w http.ResponseWriter, req *http.Request) (interface{}, error) {
+		notice, err := ps.Get(req.Context(), admindb.NoticeDescription, "en-GB")
```

Comment:
:+1:

Reply:
By the way, I'll have to change the `"en-GB"` to some variable. Not sure what though. Any ideas?

Reply:
Issue #42